### DISCUSSION TITLE
OldUnreal Mac to play Unreal Gold ?

### DISCUSSION THREAD

**Discussion by Nergus** [OldUnreal/UnrealTournamentPatches#1425] [Q&A] [answered]
Playing UT on this open source port is a bliss, but I would LOVE to play Unreal 1 too without having to go through emulation (which is actually pretty rough on this game even on great machines). Is there a way to play Unreal 1 via this port of UT or even an open source port for Unreal 1 on mac that is being prepared ?

Thank you !

**Comment by filipopo** [comment]
I think I saw somewhere that someone at least attempted to make a universal version of the Unreal 1-2 Engine for all the released games, but it's a hard task since I think the engine is a bit different for every game and the project probably died out, I would love to hear that someone got it working though

**Reply by Nergus** [reply]
From what I know every game has a bit of a different coding but Unreal Gold and UT99 both run on the same Unreal Engine 1 that also powered the original Deus Ex and Rune, so I don't think it would be as hard as to port Unreal 2 and UT2004 which run on Unreal Engine 2 (UT2004 had a Mac port that worked until Mac OS Mojave but now doesn't work anymore since it was a 32-bit app).

**Comment by stijn-volckaert** [comment, marked as answer]
We have a fully functional source port of Unreal 1, but Epic hasn't approved the release yet. Unfortunately, I have no idea if/when they will give their blessing.

**Reply by Nergus** [reply]
I doubt it will, since they straight up removed every single Unreal game from all stores. That's a real shame actually, but thanks for your reply, I hope it will get better someday. Thanks for your work ! I'm tired of having to play Unreal 1 with mediocre emulation.

**Reply by filipopo** [reply]
Is it.... available somewhere? like private releases... being shared around on discord and stuff like that

**Reply by Nergus** [reply]
Hi, it seems the team finally managed to get an agreement to release Unreal for Mac, with downloads here : https://www.macsourceports.com/game/unreal

It's still a bit rough on my side, I can't manage to launch the app on Mac OS Sierra, and the contrast levels are all over the place, but it is amazing to be able to play this game natively and in a much smoother framerate than with Windows emulation. So thank you, waiting for the next updates for an even better experience !

**Reply by stijn-volckaert** [reply]
Feel free to create GitHub issues on https://github.com/OldUnreal/Unreal-testing/issues

We likely won't get that many mac testers!
